Calendar/Get-RBASummary.ps1

```diff
@@ -220,13 +221,13 @@ function OutputRBAProcessing {
     }
 }
 
-# ToDo check if workspace settings...
+# ToDo: Future Work: Check Workspace settings...
```

Comment:
Recommend to also create an issue for this as well.

Reply:
Done
[Work Item] Improve RBA-Summary by checking WorkSpaces as well. #1774